test/unittest/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ else()
         tests/block/test_extent_list_insert_wo_block.cpp
         tests/block/test_extent_list_search.cpp
         tests/block/test_extent_list_insert_block.cpp
+        tests/block/util_block.cpp
```

Comment:
I believe header files are meant to be placed under the `unittest_sources` section.

Reply:
You're right, although it's not a header, that file should be in the `unittest_sources` section because it is contains 'helper' code rather than actual tests.